tests/conftest.py

```diff
@@ -234,3 +244,34 @@ def get_test_client(server: Sanic) -> SanicTestClient:
     test_client = server.test_client
     test_client.port = None
     return test_client
+
+
+def write_endpoint_config_to_yaml(
+    path: Path, data: Dict[Text, Any], endpoints_filename: Text = "endpoints.yml"
+) -> Path:
+    endpoints_path = path / endpoints_filename
+
+    # write endpoints config to file
+    io_utils.write_yaml_file(
+        data, endpoints_path,
+    )
+    return endpoints_path
+
+
+def random_user_uttered_event(timestamp: Optional[float] = None) -> UserUttered:
+    return UserUttered(
+        uuid.uuid4().hex,
+        timestamp=timestamp if timestamp is not None else random.random(),
+    )
+
+
+class MockMigrator(Migrator):
```

Comment:
why is this in the `conftest` module?

Reply:
because it's used in `test_exporter.py` and `test_rasa_export.py`